Docs/user_guide/getting_started.md

```diff
@@ -149,3 +149,51 @@ Browse the [User Guide](user_interface) section to find quick overview of the ap
+- Cell: Data cells are simple topological elements like lines, polygons, tetrahedra, etc.
+- Coordinate system (or coordinate frame, reference frame, space): Specified by position of origin, axis directions, and distance unit. All coordinate systems in 3D Slicer are right-handed.
+- Extension: A collection of modules that is not bundled with the core application but can be downloaded and installed using the Extensions manager.
+- [Extensions manager](extensions_manager): A software component of Slicer that allows browsing, installing, uninstalling extensions in the [Extensions catalog (also known as the Slicer app store)](http://slicer.kitware.com/midas3/slicerappstore) directly from the application.
+- [Extensions index](https://github.com/Slicer/ExtensionsIndex): A repository that contains description of each extension that the Extension catalog is built from.
+- Extent: Range of integer coordinates along 3 axes. Typically defined by 6 values, for IJK axes: `I_min`, `I_max`, `J_min`, `J_max`, `K_min`, `K_max`. Both minimum and maximum values are inclusive, therefore size of an array is `(I_max - I_min + 1)` x `(J_max - J_min + 1)` x `(K_max - K_min + 1)`.
+- Fiducial: Represents a point in 3D space. The terms originates from image-guided surgery, where "fiducial markers" are used to mark point positions.
+- Frame: One time point in a time sequence. To avoid ambiguity, this term is not used to refer to a slice of a volume.
+- Geometry: Specify location and shape of an object in 3D space.
+- Intensity: Typically refers to value of a voxel.
+- IJK: Voxel coordinate system axes. Each integer coordinate value combination corresponds to a voxel center.
+- ITK: [Insight Toolkit](https://itk.org/). Software library that Slicer uses for most image processing operations.
+- Labelmap (or labelmap volume, labelmap volume node): Volume node that has discrete (integer) voxel values. Typically each value corresponds to a specific structure or region. This allows compact representation of non-overlapping regions in a single 3D array. Most software use a single labelmap to store an image segmentation, but Slicer uses a dedicated segmentation node, which can contain multiple representations (multiple labelmaps to allow storing overlapping segments; closed surface representation for quick 3D visualization, etc.).
+- LPS: Left-posterior-superior anatomical coordinate system. Most commonly used coordinate system in medical image computing. Slicer stores all data in LPS coordinate system on disk (and converts to/from RAS when writing to or reading from disk).
+- Markups: Simple geometric objects and measurements that the user can place in viewers. [Markups module](modules/markups) can be used to create such objects. There are several types, such as fiducial list, line, curve, plane, ROI.
+- Master volume: Voxel values of this volume is used during segmentation by those effects that rely on intensity of an underlying volume.
```

Comment:
```suggestion
- Master volume: Voxel values of this volume are used during segmentation by those effects that rely on intensity of an underlying volume.
 ```

Reply:
Side comment: the term `master` is being deprecated in many contexts.  Any objection to using the term `reference` instead?  This would be consistent with dicom usage and is also makes sense that you can have multiple reference volumes for the same segmentation.

Reply:
We use "reference volume" for too many things. Most importantly, reference volume is already used for segmentation: it is the volume that specifies reference geometry (that is used for determining initial segmentation geometry and exported labelmap geometry). Maybe it could be "Guiding volume"?

We also use "master representation" for the representation that all the other representations are computed from. If anyone feels strongly about it, it could be renamed to "source" representation.

I would not replace "master" with a word that would make the meaning less clear, but if we can find better words then we could change them.

@cpinter

Reply:
"Reference volume" is heavily overloaded and used for other purposes in segmentation. We could replace the word if we find better alternative. Maybe we could use "Guide volume" (and "Source representation").

Reply:
github now creates the 'main' branch by default.  I still like 'reference' volume in the context of segmentation but I agree it's not super-specific.  'background' could be good, as could 'source' or 'input'.  We use 'input' often for non-interactive segmentations, so that could be consistent.

Reply:
>  find better alternative

Relevant read https://www.theserverside.com/opinion/Master-slave-terminology-alternatives-you-can-use-right-now

Reply:
We already use "Reference volume" for something else in segmentation (this is the volume that determines initial geometry of the segmentation and it is used when exporting the segmentation to labelmap), so we cannot use it to replace master volume.

Background volume is also used in the same context. Master volume is set as background volume by default. It could be very confusing if we started to use this for something else, too. "Source" and "Input" could be good (there is no name clash in the context of segmentation), but it would be somewhat of a regression to replace a specific term with a generic term ("input volume" is used at many places; "source volume" is not very specific either and it somewhat suggests that it is "the" source of segmentation).

Either "Guide volume" or "Input volume" sounds good to me.

@cpinter what do you think?

Reply:
> github now creates the 'main' branch by default

On a related note, I updated `slicer.org` repo to use `main` as the default branch name. See https://github.com/Slicer/slicer.org/commit/5ef131bdb8125fbe91b85e7722d35d61568ddd05

Reply:
To pick up again on this side conversation, another idea could be to use the term "Evidence" volume.

This would be consistent with the dicom Current Requested Procedure Evidence Sequence as discussed [in highdicom](https://github.com/MGHComputationalPathology/highdicom/issues/62).

Reply:
My group uses the term "Primary" instead of "Master".

Reply:
Evidence volume: Maybe DICOM will make "evidence volume" a familiar term in a few years, but for now it sounds foreign and unnecessarily abstract to me.

Primary volume: Appointing a "primary volume" at the scene level could be good. It could also be a good name for the reference volume (i.e., the _first_ selected master volume, which determines the geometry of the segmentation). However, the input volume that is currently selected in the segment editor widget is just guiding the segmentation right now - it is not permanent, distinguished role.

ITK-Snap calls it the "main image". In MITK, it is called "Patient image". They are permanently assigned to the segmentation, so you cannot simply replace them as in Slicer.

I like the "guiding volume" term (it well matches to actual role), but if it does not sound good to you then maybe we could simply use "input volume" as @pieper suggested above - at least that is very simple and already used throughout Slicer extensively.

Reply:
Weighing the options do I lean toward 'Input volume' for consistency with the rest of the app and because it is logical from the point of view of the algorithms like GrowCut.  It also doesn't imply permanence like 'primary volume' would.

I see your logic for guiding, but that has a connotation of intelligence which is wouldn't make sense here.  Perhaps 'guide' volume could work.